src/runners/handlers/jira.py

```diff
@@ -104,18 +104,25 @@ def create_jira_ticket(alert, assignee=None, custom_field=None):
     body = jira_ticket_body(alert)
 
     log.info(f'Creating new JIRA ticket for "{alert["TITLE"]}" in project {PROJECT}')
-    args_dict = {'project': PROJECT,
-                 'issuetype' : {'name': 'Story'},
-                 'summary': alert['TITLE'],
-                 'description' : body,
-                }
-    if os.environ.get('CUSTOM_FIELD_AREA'):
-        args_dict['customfield_11401'] = {'value': os.environ.get('CUSTOM_FIELD_AREA')}
-
-    new_issue = jira.create_issue(**args_dict)
-
-    if custom_field:
-        new_issue.update(fields={custom_field['id']: {'value': custom_field['value']}})
+
+    issue_params = {
+        'project': PROJECT,
+        'issuetype': {'name': 'Story'},
+        'summary': alert['TITLE'],
+        'description': body,
+    }
+
+    env_fields = os.environ.get('SA_JIRA_CUSTOM_FIELDS')
+    if env_fields or custom_field:
```

Comment:
Thanks for this change. I thought a very similar approach to make my changes more general, but then backed off thinking there may not be a need to support for those many custom field.

Reply:
We may not have more than one, but we do want to keep it extensible to Jira's outside of our main corp one.